packages/databricks-vscode/src/extension.ts

```diff
@@ -173,7 +174,17 @@ export async function activate(
         return undefined;
     }
 
-    const workspaceUri = workspace.workspaceFolders[0].uri;
+    const workspaceFolderManager = new WorkspaceFolderManager(
+        customWhenContext
+    );
+
+    context.subscriptions.push(
+        telemetry.registerCommand(
+            "databricks.selectWorkspaceFolder",
```

Comment:
We can also add it to package.json and expose it in the command palette UI

Reply:
I want to keep it hidden so that users don't mess up their projects.